### PR TITLE
archive

### PR DESCRIPTION
**Pull request by deadlyicon**

### CODE REVIEW

File: browser/components/molecules/PrrrsTable/index.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ export default class PrrrsTable extends Component {
         </td>
         {renderAdditionalCells(prrr)}
         <td>
-          <Button onClick={_ => confirmDestroyPrrr(href, prrr)} disabled={!requrestByCurrentUser}>
+          <Button onClick={_ => confirmArchivePrrr(href, prrr)} disabled={!requrestByCurrentUser}>
```
Comment (shakalee14) [inline browser/components/molecules/PrrrsTable/index.js]: 'requrestByCurrentUser' should be 'requestByCurrentUser'?

File: server/database/migrations/20170104145003_adding_arcived_at_to_pull_request_review_requests.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+exports.up = knex =>
+  knex.schema.table('pull_request_review_requests', table => {
+    table.timestamp('archived_at')
```
Comment (shakalee14) [inline server/database/migrations/20170104145003_adding_arcived_at_to_pull_request_review_requests.js]: does this also need to start off as null? will this automatically mark it as null in the archived_at column? 
.defaultTo(null)

File: server/queries/index.js [hunk 3]
```diff
@@ -31,6 +31,7 @@ export default class Queries {
       .select('*')
       .from('pull_request_review_requests')
       .orderBy('created_at', 'asc')
+      .where('archived_at', null)
```
Comment (shakalee14) [inline server/queries/index.js]: we add a where statement here too for prrrs that have a null value at 'created_at'.

File: server/database/migrations/20170104145003_adding_arcived_at_to_pull_request_review_requests.js (hunk 2 above)
Comment (lg-jared-test-account-1) [inline server/database/migrations/20170104145003_adding_arcived_at_to_pull_request_review_requests.js]: I think `defaultTo(null)` is implied.

**Pull request merged**